R/module_teal.R

```diff
@@ -160,18 +160,19 @@ srv_teal <- function(id, modules, teal_data_rv, filter = teal_slices()) {
       )
     }
 
-    env <- environment()
     datasets_reactive <- eventReactive(teal_data_rv(), {
-      env$progress <- shiny::Progress$new(session)
-      env$progress$set(0.25, message = "Setting data")
-
+      progress_d <- Progress$new(
```

Comment:
what if we use names `progress_data` and `progress_modules`
```suggestion
      progress_data <- Progress$new(
```

Reply:
Then one line will run over the 120 character limit and another `# nolint` exception will have to be added.

Reply:
I think we can leave with that :)

Reply:
done

Reply:
@chlebowa can we get back to progress_data from the initial review?

Reply:
Yup, sorry about that.